### PR TITLE
Add circle and disk to glossary.md

### PR DESCRIPTION
**Pull request by ovynnej**
## Description

This PR adds the terms "circle" and "disk" to glossary.md to address issue #4455. 

Thank you!

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: doc/source/glossary.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ channel
     represent color or alpha information, but may be any generic batch
     dimension over which to operate.
 
+circle
+    The perimeter of a {term}`disk`.
```
Comment (grlee77) [inline doc/source/glossary.md:26]: nice, I was not familiar with this use of term to link terms in the glossary!

**Review by grlee77 — APPROVED** [review]
Thanks @ovynnej. This looks good, but the overall rendering of the page looks off to me. Not sure if you are interested in helping address that here or in a follow-up PR?

See how the docs for contour/iso-valued contour/contrast render poorly here:

glossary

The biggest problem seems to be with the attempt to link multiple terms to the same description. Perhaps two terms sharing an entry should get separate entries like this?:

```
contour
    <existing description here>
iso-valued contour
    see {term}`contour`
```

Hopefully that would cause the glossary to render more clearly. Do you have other ideas how that could be addressed?

**Comment by ovynnej** [comment]
Thank you, @grlee77! I like your idea of separating entries for terms that have the same description in the way that you described. Yes, I would like to work on this in a follow-up PR. Would this be the correct process?

1. Wait for you or someone else to merge edit-glossary to main (since I'm not authorized to do that)
2. Make my edits, commit, and push to this same edit-glossary branch
3. Make another PR to merge this edit-glossary branch to main

**Comment by jni** [comment]
@ovynnej thank you for your contribution!

I would suggest the following procedure, which will work in general, now that we've merged your changes:

- locally, switch back to the main branch.
- get the latest main from this repo. For me this is the command `git pull upstream main`, but that depends on having set the upstream correctly. You can also go to [your fork](https://github.com/ovynnej/scikit-image) and click on "fetch upstream" near the top right, in order to update your own clone. Then you can `git pull origin main`, locally, again while in the main branch locally.
- start a new branch (eg add-glossary-term-links) and make a new pull request from that branch.

Does that make sense?

**Comment by ovynnej** [comment]
Thank you for the helpful explanation, @jni! I will do this.

**Pull request merged**